packages/components/src/components/textarea/textarea.stories.ts

```diff
@@ -30,400 +32,247 @@ export default {
+        spellcheck
+      ></sd-textarea>
+    </div>
+  `
 };
 
 /**
- * 1. You can use the `setCustomValidity` method to set a custom validation message. This will override any native validation messages.
- * 2. Set an empty string to clear the custom validity and make the input valid.
- * 3. To show the validation message, call the `reportValidity` method. Originally this would show a native validation bubble, but we show the error messages inline.
+ * Use the `help-text` slot to add help text.
  */
 
-export const setCustomValidity = {
-  parameters: {
-    chromatic: { disableSnapshot: true }
-  },
-  render: () => {
-    return html`
-      <!-- block submit and show alert instead -->
-      <form id="validationForm" class="flex flex-col gap-2">
-        <sd-textarea id="custom-input" label="Input" style-on-valid></sd-textarea>
-        <div>
-          <sd-button type="submit">Submit</sd-button>
-          <sd-button id="error-button" variant="secondary">Set custom error</sd-button>
-          <sd-button id="success-button" variant="secondary">Set success</sd-button>
-          <sd-button type="reset" variant="secondary">Reset</sd-button>
-        </div>
-      </form>
-      <script type="module">
-        // Wait for custom elements to be defined
-        await Promise.all([customElements.whenDefined('sd-input'), customElements.whenDefined('sd-button')]).then(
-          () => {
-            const form = document.getElementById('validationForm');
-            const input = document.getElementById('custom-input');
-            const setErrorButton = document.getElementById('error-button');
-            const setSuccessButton = document.getElementById('success-button');
-
-            // Initial error
-            const errorMessage = \`This is an initial custom error (\${new Date().toLocaleTimeString()})\`;
-            input.setCustomValidity(errorMessage);
-            input.reportValidity();
-
-            // Show error message
-            setErrorButton.addEventListener('click', () => {
-              const errorMessage = \`This is a new custom error (\${new Date().toLocaleTimeString()})\`;
-              input.setCustomValidity(errorMessage);
-              input.reportValidity();
-            });
-
-            // Show success message
-            setSuccessButton.addEventListener('click', () => {
-              input.setCustomValidity(''); // Clear custom validity
-              input.reportValidity();
-            });
+export const HelpText = {
+  name: 'Help Text',
+  render: () => html`
+    <div class="w-[500px]">
+      <sd-textarea size="lg" label="Label" help-text="Help text example" rows="4"></sd-textarea>
+    </div>
+  `
+};
 
-            form.addEventListener('submit', event => {
-              event.preventDefault();
-              alert('All fields are valid!');
-            });
-          }
-        );
-      </script>
-    `;
-  }
+/**
+ * Use the `required` attribute to mark the textarea as required.
```

Comment:
![image](https://github.com/user-attachments/assets/a5370188-ef05-4366-90f8-a11b947a5e1f)

Use the ”required” attribute to mark the element as required. This can be used for form validation purposes.